conf/homeserver.yaml

```diff
@@ -1358,7 +1356,10 @@ registration_shared_secret: "__REGISTRATION_SHARED_SECRET__"
 # by the Matrix Identity Service API specification:
 # https://matrix.org/docs/spec/identity_service/latest
 #
-account_threepid_delegates:
+# As email delegates is managed by the synapse server itself this email section is 
+# not necessary but msisdn format is still composed by msisdn: <value> on a new line
+account_threepid_delegates: 
+    __ACCOUNT_THREEPID_DELEGATES_MSISDN__
```

Comment:
The error might come from the following:

When you do `ynh_add_config` `__ACCOUNT_THREEPID_DELEGATES_MSISDN__` will correctly be replaced by `#email: `

But when the config panel writes or reads the file with bind, it will read/write just after `account_threepid_delegates: ` (on the same line). 

So if we want to read/write in the next line, we should write yet another setter/getter with your magic sed :D

To be tested